lnst/Tests/TrafficControl.py

```diff
@@ -31,18 +35,44 @@ def run(self) -> bool:
 
     async def run_instances(self) -> list[dict]:
         tc_exec = shutil.which("tc")
+        cpu_bind = self.params.get("cpu_bind")
+        cpu_bind_gen: Iterator[Optional[list[int]]]
+        if cpu_bind is not None:
+            policy = self.params.cpu_bind_policy
+            if policy == "round-robin":
+                cpu_bind_gen = ([cpu] for cpu in itertools.cycle(cpu_bind))
+            else:
+                cpu_bind_gen = itertools.repeat(cpu_bind)
+        else:
+            cpu_bind_gen = itertools.repeat(None)
+
         instances = [
-            self.run_tc(tc_exec, bf)
+            self.run_tc(tc_exec, bf, cpu_bind=next(cpu_bind_gen))
             for bf in self.params.batchfiles
         ]
         results = await asyncio.gather(*instances)
         return results
 
-    async def run_tc(self, tc_exec: str, batchfile: str) -> dict[str]:
+    async def run_tc(
+        self,
+        tc_exec: str,
+        batchfile: str,
+        cpu_bind: Optional[list[int]] = None,
+    ) -> dict[str]:
+        args = [tc_exec, "-b", batchfile]
+        if cpu_bind is not None:
+            args = [
+                "/usr/bin/taskset",
+                "-c",
+                ",".join(map(str, cpu_bind)),
+                *args,
+            ]
+
         start_timestamp = time.time()
         start_time = time.perf_counter()
+        logging.debug(args)
```

Comment:
Can we make this into something more descriptive. Such debug log entry would be quite hard to find in the code.

Reply:
Fixed (hopefully).